website/docs/support/product-behaviour.md

```diff
@@ -0,0 +1,9 @@
+# Product behavior
+
+Below is a list of possibly unexpected behaviors/limitations of the project.
```

Comment:
Is the difference between these notes and the known-issues page significant enough to create another page?  I worry we're splitting ownership between things where the owner is unclear.  Not a big deal when adding details, but possibly confusing for end users who go looking for this sort of info.

Could we get deliver the same info in a better way, and in a single page, if we update the known issues page with more structure?

Reply:
> Could we get deliver the same info in a better way, and in a single page, if we update the known issues page with more structure?

The suggestion for a separate page came through @vkamra , but having it in a single page could also work out I guess. The primary difference here is that these are things which might be "unexpected" behaviors but the `known_issues` page is actual issues we indent to fix(if possible).

Reply:
We could try doing this on the same page as `Known Issues` - and see if that is cleaner. 

How about we rename `Known Issues` to `Known Issues and Limitations` -> and add the structure @meain  is proposing below.

Reply:
@ryanfkeepers @vkamra I've combined both the pages. There were a few items in "known issues" which would better fit under "limitations".